### PR TITLE
feat: [#188049417] Bubble investNewark fundings to top of For You column

### PR DESCRIPTION
**Pull request by SirSamuelJoseph**
## Description

Switches the order of fundings and certifications in the For You column if the user signed up through an investNewark link. Also moves fundings from investNewark to the top of the fundings list for those users

### Ticket

This pull request resolves [#188049417](https://www.pivotaltracker.com/story/show/188049417).

### Steps to Test

Signup through this link: http://localhost:3000/onboarding?businessMunicipality=Newark&utm_source=investNewark
Sign up as an Oscar or a Poppy that advances to the up and running stage.
You should see fundings over certifications and the two investNewark fundings at the top of the funding list. The sorting of the rest of the list should remain unchanged.

Creating an account through any other means should not see behavior change

## Code author checklist

- [x] I have rebased this branch from the latest main branch
- [x] I have performed a self-review of my code
- [x] I have created and/or updated relevant documentation (EX: [Engineering Reference/FAQ](https://docs.google.com/document/d/1X4iWSGmBZdHYZ0jGqkwTR3_yyyqI_TiiJptfc8pi9Po)), if necessary
- [x] I have not used any relative imports
- [x] I have checked for and removed instances of unused config from CMS
- [x] I have pruned any instances of unused code
- [x] I have not added any markdown to labels, titles and button text in config
- [x] If I added/updated any values in `userData` (including `profileData`, `formationData` etc), then I added a new migration file
- [x] If I added any new collections to the CMS config, then I updated the search tool and `cmsCollections.ts` (see [CMS Additions in Engineering Reference/FAQ](https://docs.google.com/document/d/1X4iWSGmBZdHYZ0jGqkwTR3_yyyqI_TiiJptfc8pi9Po/edit#heading=h.fu5jdsrcqxbh))
- [x] I have updated relevant `.env` values in both `.env-template` and in Bitwarden

### CODE REVIEW

**Review by AnJuHyppolite — APPROVED** [review]
LGTM

**Review by cbrunefearless — APPROVED** [review]
lgtm

**Review by seidior — APPROVED** [review]
Looks good, Sam! Just a few minor comments here, but overall I think this will set us up well for other userData-based modifications to the sidebar.

File: web/src/lib/domain-logic/sidebarCardsHelpers.ts [hunk 1]
```diff
@@ -44,8 +44,8 @@ export const getVisibleCertifications = (
     return !business?.preferences.hiddenCertificationIds.includes(it.id);
   });
 };
-export const sortFundings = (fundings: Funding[]): Funding[] => {
-  return fundings.sort((a, b) => {
+export const sortFundings = (fundings: Funding[], userData?: UserData): Funding[] => {
```
[the remark this answers is not in the document]
Comment (SirSamuelJoseph) [reply web/src/lib/domain-logic/sidebarCardsHelpers.ts:47]: Addressed in: https://github.com/newjersey/navigator.business.nj.gov/pull/8435

File: web/src/lib/domain-logic/sidebarCardsHelpers.ts [hunk 2]
```diff
@@ -44,8 +44,8 @@ export const getVisibleCertifications = (
...
+  const initialSorting = fundings.sort((a, b) => {
     const nameA = a.name.toUpperCase(); // ignore upper and lowercase
     const nameB = b.name.toUpperCase();
     if (FundingStatusOrder[a.status] < FundingStatusOrder[b.status]) {
```
[the remark this answers is not in the document]
Comment (SirSamuelJoseph) [reply web/src/lib/domain-logic/sidebarCardsHelpers.ts:51]: Addressed in: https://github.com/newjersey/navigator.business.nj.gov/pull/8435

**Pull request merged**